src/routes/UnsyncedRecordsRoute.tsx

```diff
@@ -0,0 +1,854 @@
+				date: truckInYardHistory.enteredDateTime,
+				details: {
+					"Truck ID": truckInYardHistory.truck?.truckID,
+					Hauler: truckInYardHistory.truck?.hauler.haulerID + " - " + truckInYardHistory.truck?.hauler.haulerName,
+					"Location Name": truckInYardHistory.location?.locationName,
+					"Entered Date": utcToLocalDatetime(truckInYardHistory.enteredDateTime.toString(), "MM/dd/yy hh:mm a"),
+
+					ID: truckInYardHistory.id,
+					"Truck Key": truckInYardHistory.truck?.truckKey,
+					"Hauler Key": truckInYardHistory.truck?.hauler.haulerKey,
+					"Location Key": truckInYardHistory.location?.locationKey,
+				},
+				syncedError: truckInYardHistory.syncedError ?? "",
+				id: truckInYardHistory.id,
+			} as UnsyncedRecords;
+
+			truckInYardHistory.loaderID && Object.assign(record.details, { "Loader ID": truckInYardHistory.loaderID });
+			truckInYardHistory.targetWeight &&
+				Object.assign(record.details, { "Target Weight": truckInYardHistory.targetWeight });
+			truckInYardHistory.request?.requestKey &&
+				Object.assign(record.details, { Request: truckInYardHistory.request.info });
+			if (truckInYardHistory.orderProduct?.orderProductKey) {
+				Object.assign(record.details, {
+					"In-Yard Customer":
+						truckInYardHistory.orderProduct?.order.customer.customerID +
+						" - " +
+						truckInYardHistory.orderProduct?.order.customer.customerName,
+				});
+				Object.assign(record.details, {
+					"In-Yard Order":
+						truckInYardHistory.orderProduct?.order.orderNumber +
+						" - " +
+						truckInYardHistory.orderProduct?.order.description,
+				});
+				Object.assign(record.details, {
+					"In-Yard Product":
+						truckInYardHistory.orderProduct?.product.productID +
+						" - " +
+						truckInYardHistory.orderProduct?.product.productDescription,
+				});
+			}
+
+			truckInYardHistory.request?.requestKey &&
+				Object.assign(record.details, { "Request Key": truckInYardHistory.request.requestKey });
+			if (truckInYardHistory.orderProduct?.orderProductKey) {
+				Object.assign(record.details, {
+					"In-Yard Customer Key": truckInYardHistory.orderProduct?.order.customer.customerKey,
+				});
+				Object.assign(record.details, {
+					"In-Yard Order Key": truckInYardHistory.orderProduct?.order.orderKey,
+				});
+				Object.assign(record.details, {
+					"In-Yard OrderProduct Key": truckInYardHistory.orderProduct?.orderProductKey,
+				});
+				Object.assign(record.details, {
+					"In-Yard Product Key": truckInYardHistory.orderProduct?.product.productKey,
+				});
+			}
+
+			unsyncedRecords.push(record);
+		});
+
+		return unsyncedRecords.sort((a, b) => compareAsc(new Date(a.date), new Date(b.date)));
+	}, [unsyncedTicketsQuery, unsyncedTrucksQuery, unsyncedTrucksInYardQuery, unsyncedTruckInYardHistoryQuery]);
+
+	const numberOfErrors = unsyncedRecordsData
+		.map((r) => {
+			if (r.syncedError != "" && r.syncedError != null) {
+				return r.syncedError;
+			}
+			return;
+		})
+		.filter((r) => r).length;
```

Comment:
`.map()` is probably unnecessary here, you can achieve correct length with chained `.filter()` function
```suggestion
		.filter((r) => r.syncedError)).length
```